toolchain/check/context.cpp

```diff
@@ -118,6 +118,38 @@ auto Context::FinishInst(SemIR::InstId inst_id, SemIR::Inst inst) -> void {
   }
 }
 
+// Returns whether a parse node associated with an instruction of kind
+// `src_kind` is known to be usable as the location of an instruction of kind
+// `target_kind`. This may have false negatives for cases that don't occur in
+// practice.
```

Comment:
By "don't occur in practice", is this just noting how it's really only handling equality and the `ImportDecl` special-case? If so, maybe this could instead say something like:

```suggestion
// `target_kind`. This may have false negatives when InstKinds don't match,
// but that only occurs for ImportDecl/Namespace in practice.
```

A different option you might consider is renaming this to something like "IsCompatibleImportedInstKind", then it's not a false negative, it's just that ImportDecl/Namespace is the only case where we twiddle instruction kinds IIUC.

Reply:
Renamed.